specs/phase0/p2p-interface.md

```diff
@@ -745,10 +747,27 @@ The request MUST be encoded as an SSZ-container.
 The response MUST consist of zero or more `response_chunk`.
 Each _successful_ `response_chunk` MUST contain a single `SignedBeaconBlock` payload.
 
-Clients MUST keep a record of signed blocks seen since the start of the weak subjectivity period
-and MUST support serving requests of blocks up to their own `head_block_root`.
+Clients MUST keep a record of signed blocks seen on the epoch range
+`[max(GENESIS_EPOCH, current_epoch - MIN_EPOCHS_FOR_BLOCK_REQUESTS), current_epoch]`
+where `current_epoch` is defined by the current wall-clock time,
+and clients MUST support serving requests of blocks on this range.
+
+Peers are *repeatedly* unable to reply to Block requests within the
```

Comment:
```suggestion
Peers that are unable to reply to Block requests within the
```

`repeatedly` opens up for interpretation in a way that makes the whole statement ambiguous - there's already enough ambiguity in `MAY` to cover for repeatedly also.